### ISSUE TITLE
firsterrorwindow

### ISSUE THREAD

**Issue by 12pixels0** [spyder-ide/spyder#12751]
### What steps will reproduce the problem?

<h1> firsterrorwindow </h1>
<p>No,thanks!</p>
<p>goodbye!</p>

### Traceback
```python-traceback
Exception in comms call get_cwd:
  File "E:\program\Anaconda\lib\site-packages\spyder_kernels\comms\commbase.py", line 342, in _handle_remote_call
    buffer['call_kwargs'])
  File "E:\program\Anaconda\lib\site-packages\spyder_kernels\comms\commbase.py", line 353, in _remote_callback
    *call_args, **call_kwargs)
  File "E:\program\Anaconda\lib\site-packages\spyder_kernels\console\kernel.py", line 356, in get_cwd
    return os.getcwd()
TypeError: 'int' object is not callable
WARNING:spyder.widgets.github.backend:Failed to send bug report on Github. response={'code': 401, 'json': {'message': 'Bad credentials', 'documentation_url': 'https://developer.github.com/v3'}}
WARNING:spyder.widgets.github.backend:Failed to send bug report on Github. response={'code': 401, 'json': {'message': 'Bad credentials', 'documentation_url': 'https://developer.github.com/v3'}}
WARNING:spyder.widgets.github.backend:Failed to send bug report on Github. response={'code': 401, 'json': {'message': 'Bad credentials', 'documentation_url': 'https://developer.github.com/v3'}}
WARNING:spyder.widgets.github.backend:Failed to send bug report on Github. response={'code': 401, 'json': {'message': 'Bad credentials', 'documentation_url': 'https://developer.github.com/v3'}}
```

## Versions

* Spyder version: 4.0.1 
* Python version: 3.7.6
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```
atomicwrites >=1.2.0         :  1.3.0 (OK)
chardet >=2.0.0              :  3.0.4 (OK)
cloudpickle >=0.5.0          :  1.3.0 (OK)
diff_match_patch >=20181111  :  20181111 (OK)
intervaltree                 :  None (OK)
IPython >=4.0                :  7.12.0 (OK)
jedi =0.14.1                 :  0.14.1 (OK)
nbconvert >=4.0              :  5.6.1 (OK)
numpydoc >=0.6.0             :  0.9.2 (OK)
pexpect >=4.4.0              :  4.8.0 (OK)
pickleshare >=0.4            :  0.7.5 (OK)
psutil >=0.3                 :  5.6.7 (OK)
pygments >=2.0               :  2.5.2 (OK)
pylint >=0.25                :  2.4.4 (OK)
pyls >=0.31.2;<0.32.0        :  0.31.7 (OK)
zmq >=17                     :  18.1.1 (OK)
qdarkstyle >=2.7             :  2.8 (OK)
qtawesome >=0.5.7            :  0.6.1 (OK)
qtconsole >=4.6.0            :  4.6.0 (OK)
qtpy >=1.5.0                 :  1.9.0 (OK)
rtree >=0.8.3                :  0.9.3 (OK)
sphinx >=0.6.6               :  2.4.0 (OK)
spyder_kernels >=1.8.1;<2.0.0:  1.8.1 (OK)
watchdog                     :  None (OK)
cython >=0.21                :  0.29.15 (OK)
matplotlib >=2.0.0           :  3.1.3 (OK)
numpy >=1.7                  :  1.18.1 (OK)
pandas >=0.13.1              :  1.0.1 (OK)
scipy >=0.17.0               :  1.4.1 (OK)
sympy >=0.7.3                :  1.5.1 (OK)
```

**Comment by juanis2112** [comment]
Hi @12pixels0 can you try updating spyder to the latest version 4.1.3 and verify if the error persists?
If it does, can you please explain the steps that reproduce the error?

**Comment by 12pixels0** [comment]
No,thanks.this error is my delete os's getcwd after this window just show.
<I'm a chinese user,English to me is a little hard,if anywhere is wrong,please answer me,thanks.&gt;
 



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Juanita Gomez"<notifications@github.com&gt;; 
发送时间: 2020年5月16日(星期六) 凌晨2:16
收件人: "spyder-ide/spyder"<spyder@noreply.github.com&gt;; 
抄送: "陈先权"<2750430086@qq.com&gt;; "Mention"<mention@noreply.github.com&gt;; 
主题: Re: [spyder-ide/spyder] firsterrorwindow (#12751)





 
Hi @12pixels0 can you try updating spyder to the latest version 4.1.3 and verify if the error persists?
 If it does, can you please explain the steps that reproduce the error?

**Comment by juanis2112** [comment]
Type the following in Anaconda prompt for updating spyder.
conda update anaconda
conda install spyder=4.1.3

**Comment by 12pixels0** [comment]
OK.


 



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Juanita Gomez"<notifications@github.com&gt;; 
发送时间: 2020年5月26日(星期二) 上午8:25
收件人: "spyder-ide/spyder"<spyder@noreply.github.com&gt;; 
抄送: "陈先权"<2750430086@qq.com&gt;; "Mention"<mention@noreply.github.com&gt;; 
主题: Re: [spyder-ide/spyder] firsterrorwindow (#12751)





 
Closed #12751.
